src/main/java/frc/robot/Constants.java

```diff
@@ -191,23 +200,23 @@ public final class DoryCameras {
       public static final Transform3d frontLeftFromRobot =
           new Transform3d(
               new Translation3d(
-                  Units.inchesToMeters(11), Units.inchesToMeters(7), Units.inchesToMeters(4)),
-              new Rotation3d(Units.degreesToRadians(180), Units.degreesToRadians(-30), 0.0));
+                  Units.inchesToMeters(11), Units.inchesToMeters(-7), Units.inchesToMeters(4)),
```

Comment:
This negation on y is still odd to me that it's only on Dory front cameras and not on the back cameras or Nautilus.

I would leave a comment indicating that this is unexpected behavior so it's easy to notice in the future.